tests/helpers.py

```diff
@@ -121,3 +124,61 @@ def create_mock_agent(
         return agent
 
     raise ValueError("No valid ifi was provided to create_mock_agent")
+
+
+def mock_statement(
```

Comment:
I'm fine with this! We could ask @quitterie-lcs @jmaupetit

Reply:
Sounds good for me too